Replaces the names of KOI8 fonts with those in scaleble-cyrfonts

See: https://patches.ubuntu.com/by-release/extracted/ubuntu/a/a2ps/1:4.13c~rc5-1/03_koi8_fonts_debian.dpatch

## 03_koi8_fonts_debian.dpatch by Masayuki Hatta <mhatta@debian.org>

Index: b/encoding/koi8.edf
===================================================================
--- a/encoding/koi8.edf
+++ b/encoding/koi8.edf
@@ -1,6 +1,7 @@
 # Description of the KOI8 encoding
 # Copyright (c) 1988, 89, 90, 91, 92, 93 Miguel Santana
 # Copyright (c) 1995, 96, 97, 98 Akim Demaille, Miguel Santana
+# Copyright (c) 2002 Vyacheslav E. Sidin
 #
 
 #
@@ -27,27 +28,26 @@
 Name: 	KOI8
 
 Documentation
-KOI-8 (+Ëë) is a subset of ISO-IR-111 that can be used in Serbia, Belarus
+KOI-8 (+ª³«³) is a subset of ISO-IR-111 that can be used in Serbia, Belarus
 etc.
 EndDocumentation
 
-# Regular fonts don't support Cyrillic glyphs
-Default: 	CourierISOC
+Default: 	Courier
 
-Substitute: 	Courier			CourierISOC
-Substitute: 	Courier-Bold		CourierISOC-Bold
-Substitute: 	Courier-BoldOblique	CourierISOC-BoldOblique
-Substitute: 	Courier-Oblique		CourierISOC-Oblique
-
-Substitute: 	Times-Roman		College
-Substitute: 	Times-Bold		College-Bold
-Substitute: 	Times-Italic		College-Italic
-Substitute: 	Times-BoldItalic	College-Italic
-
-Substitute: 	Helvetica		Textbook
-Substitute: 	Helvetica-Bold		Textbook-Bold
-Substitute: 	Helvetica-Oblique	Textbook-Italic
-Substitute: 	Helvetica-BoldOblique	Textbook-Italic
+Substitute: 	Courier			NimbusMonCYR-Regu
+Substitute: 	Courier-Bold		NimbusMonCYR-Bold
+Substitute: 	Courier-BoldOblique	NimbusMonCYR-BoldObli
+Substitute: 	Courier-Oblique		NimbusMonCYR-ReguObli
+
+Substitute: 	Times-Roman		NimbusRomanCYR-Regu
+Substitute: 	Times-Bold		NimbusRomanCYR-Medi
+Substitute: 	Times-Italic		NimbusRomanCYR-ReguItal
+Substitute: 	Times-BoldItalic	NimbusRomanCYR-MediItal
+
+Substitute: 	Helvetica		NimbusSansCYR-Regu
+Substitute: 	Helvetica-Bold		NimbusSansCYR-Bold
+Substitute: 	Helvetica-Oblique	NimbusSansCYR-ReguObli
+Substitute: 	Helvetica-BoldOblique	NimbusSansCYR-BoldObli
 
 Vector:
 .notdef .notdef .notdef .notdef .notdef .notdef .notdef .notdef
